pkg/sync/handler.go

```diff
@@ -103,22 +103,35 @@ func (h *HTTPHandler) DownloadFromURL(ctx context.Context, url, filePath string,
 
 // IdleTimeoutCopy relies on ctx of the reader/src or a separate timer to interrupt the processing.
 func IdleTimeoutCopy(ctx context.Context, cancel context.CancelFunc, src io.ReadCloser, dst io.WriteSeeker, updater ProgressUpdater) (copied int64, err error) {
-	writeSeekCh := make(chan int64)
+	writeSeekCh := make(chan int64, 100)
```

Comment:
Curious. Why add `100` here?

Reply:
No specific reason. The main purpose here is to make this channel a buffered one.

Reply:
I see. Thank you.